projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -165,6 +205,26 @@ import { scoreboardFeatureKey, scoreboardReducer } from './reducers/scoreboard.r
 export class ScoreboardModule {}
 </code-example>
 
+### Using the Standalone API
```

Comment:
Should we also document using standalone with `createFeature`?
https://ngrx.io/guide/store/feature-creators

Reply:
Good catch